### PR TITLE
Show Snackbar On Error

### PR DESCRIPTION
**Pull request by buda1994**

### CODE REVIEW

File: src/Grid/DataGrid.tsx [hunk 1]
```diff
@@ -105,13 +105,19 @@ class DataGrid extends React.Component <IProps & WithStyles<keyof typeof styleCl
           searchText: tbResponse.SearchText || '',
           totalRecordCount: tbResponse.TotalRecordCount || 0
         });
+      }, (error: any) => {
+        this.props.errorHandler ? this.props.errorHandler(error) : this.throwError(error);
```
Comment (geoperez) [inline src/Grid/DataGrid.tsx]: Why do you need to add a method to perform a throw?

File: test/DataGrid.spec.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import GridHeader from '../src/Grid/GridHeader';
 import Paginator from '../src/Grid/Paginator';
 import RemoteDataSource from '../src/Grid/RemoteDataSource';
 import { validColumnsSample } from './utils/columns';
-import data from './utils/data';
+import { data } from './utils/data';
```
Comment (geoperez) [inline test/DataGrid.spec.tsx]: Why did you modify the spec file?

File: src/Grid/DataGrid.tsx [hunk 3]
```diff
@@ -65,9 +65,9 @@ interface IProps {
   showTopPager?: boolean;
   showPrintButton?: boolean;
   showExportButton?: boolean;
+  errorHandler?(error: any): any;
```
Comment (kadosh) [inline src/Grid/DataGrid.tsx]: I don't like errorHandler... it should be "onError"

File: test/DataGrid.spec.tsx (hunk 2 above)
Comment (buda1994) [reply test/DataGrid.spec.tsx]: The tests were failing

File: src/Grid/DataGrid.tsx (hunk 1 above)
Comment (kadosh) [inline src/Grid/DataGrid.tsx]: Doesn't make any sense to have a method to just throw the error... You were trying to use the ternary operator in an invalid way....

**Pull request merged**